starters/vue3-apollo-quasar/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= productName %></title>
+    <title>Vue 3, Quasar & Apollo Starter kit</title>
```

Comment:
This should take the value on the left so when the CLI runs it's replaced with the new project name.

```suggestion
    <title><%= productName %></title>
```